src/OrchardCore/OrchardCore.Shells.Azure/Extensions/BlobShellsOrchardCoreBuilderExtensions.cs

```diff
@@ -27,7 +28,7 @@ public static OrchardCoreBuilder AddAzureShellsConfiguration(this OrchardCoreBui
             services.AddSingleton<IShellsFileStore>(sp =>
             {
                 var configuration = sp.GetRequiredService<IConfiguration>();
-                var blobOptions = configuration.GetSection("OrchardCore:OrchardCore.Shells.Azure").Get<BlobShellStorageOptions>();
+                var blobOptions = configuration.GetSectionCompat("OrchardCore:OrchardCore_Shells_Azure").Get<BlobShellStorageOptions>();
```

Comment:
We need the same in `DatabaseShellConfigurationSources.cs` and in `DatabaseShellsSettingsSources`